control/tests/freqresp_test.py

```diff
@@ -433,9 +433,9 @@ def test_dcgain_consistency():
     np.testing.assert_equal(
         sys_ss(0j, warn_infinite=False), complex(np.inf, np.nan))
     np.testing.assert_equal(
-        sys_tf.dcgain(warn_infinite=False), complex(np.inf, np.nan))
+        sys_tf.dcgain(warn_infinite=False), np.inf)
```

Comment:
I think the `warn_infinite` is redundant here since `dcgain` does that by default.  OK to leave, but it isn't testing anything.

This also occurs several times below.